react/features/base/react/components/native/Header.js

```diff
@@ -2,8 +2,13 @@
 
 import React, { Component, type Node } from 'react';
 import { Platform, SafeAreaView, StatusBar, View } from 'react-native';
+import { connect } from 'react-redux';
 
-import styles, { HEADER_PADDING, STATUSBAR_COLOR } from './styles';
+import { ColorSchemeRegistry } from '../../../color-scheme';
+
+// import { isDarkColor } from '../../../styles';
```

Comment:
Debug leftover?

Reply:
Nah, I just forgot to finish the implementation of the color luminance feature. Thanks!